### PR TITLE
Functionality for reading HP and SE data

### PR DESCRIPTION
**Pull request by skymandr**
Functionality for reading HotPixel and SingleEvent data from S3. Also bumps versions of input and output buckets.

Waiting for new instrument data, otherwise I think these are all changes needed for reprocessing.

### CODE REVIEW

File: src/mats_l1_processing/instrument.py [hunk 1]
```diff
@@ -324,18 +324,23 @@ def __init__(self, channel, calibration_file):
 
         # single event correction
         filename = calibration_data['hot_pixels']['single_events']
-        file_conn = sqlite.connect(filename)
-        self.single_event = sqlite.connect(':memory:')
-        file_conn.backup(self.single_event)
-        file_conn.close()
-
+        if MATS_SQLITE_S3:
```
Comment (joakimmoller) [inline src/mats_l1_processing/instrument.py:327]: Does not really matter, but you could move the logic to "sqlite_s3" and let that module work as a "gateway" - returning the single_event data (regardless where it came from).

**Comment by joakimmoller** [comment]
OK!

File: src/mats_l1_processing/instrument.py (hunk 1 above)
Comment (skymandr) [reply src/mats_l1_processing/instrument.py:327]: That might be neater – would hide the magic from the main code. I'll have a look with future possible patterns in mind.

**Comment by skymandr** [comment]
Superseded by #203

**Pull request closed without merging**